build-system/pr-check/checks.js

```diff
@@ -55,6 +55,7 @@ async function main() {
     timedExecOrDie('gulp caches-json');
     timedExecOrDie('gulp json-syntax');
     timedExecOrDie('gulp dev-dashboard-tests');
+    timedExecOrDie('gulp check-owners');
```

Comment:
Nit/suggestion for a future PR: Might be worth defining `gulp(task) { timedExecOrDie(`gulp ${task}`); }` for convenience in this file

Reply:
Some tasks use custom flags, so this pattern won't always work. Besides I think it's clearer to explicitly spell out the entire task command.